allennlp/common/util.py

```diff
@@ -187,6 +188,14 @@ def get_spacy_model(spacy_model_name: str, pos_tags: bool, parse: bool, ner: boo
     keyed by the options we used to create the spacy model, so any particular configuration only
     gets loaded once.
     """
+
+    # Check if spacy models are available.  If not, install them.
+    try:
+        import en_core_web_sm
```

Comment:
I don't think this is quite right, don't we want to import whatever the model name they said to load is ?

Reply:
you're completely right ;-)